app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1075,6 +1089,16 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
 
         private const val AUTHENTICATION_DIALOG_TAG = "AUTH_DIALOG_TAG"
 
+        private const val MIN_PROGRESS = 10
```

Comment:
Check my comment suggesting extracting animation code out of this fragment. For example, checking this companion object is easy to detect that we have multiple `const` very related to animations + key-value `const` `Bundle` related, which are different things. I think this is also an indicator to split things, as they represent different responsabilities.

Reply:
👍 Animations are now created in a new helper class (some of them!)